core/src/Cuda/Kokkos_CudaSpace.cpp

```diff
@@ -164,9 +166,11 @@ bool CudaUVMSpace::available() {
 
 /*--------------------------------------------------------------------------*/
 
+#ifdef KOKKOS_ENABLE_DEPRECATED_CODE_3
```

Comment:
What is the deal with this one?  It was not marked as deprecated before.

Reply:
It was marked as deprecated in the header file already, see https://github.com/kokkos/kokkos/pull/4106/files/137a6ceeb5ddc68eb9b2109442e380451f731f05#diff-35c830ab7d0949fbded2b207d0f7af455d59a69948ed183687fbf3be3f342b10R159.